addon/components/ember-popper-base.js

```diff
@@ -83,12 +74,21 @@ export default class EmberPopperBase extends Component {
   @type('boolean')
   renderInPlace = false
 
+  /**
+   * The popper element needs to be moved higher in the DOM tree to avoid z-index issues.
+   * See the block-comment in the template for more details. `.ember-application` is applied
+   * to the root element of the ember app by default, so we move it up to there.
+   */
+  @argument
+  @type(unionOf('string', Element))
+  popperContainer = '.ember-application'
```

Comment:
Was there a particular reason for moving this out of alphabetical order? #bikeshed

Reply:
ah, didn't realize they were alphabetical, I was just trying to put the selectors together. Will update to put them the other way since it does both